flight_club/sessions/views.py

```diff
@@ -19,65 +19,253 @@
 from flight_club.models.models import Beer, Session
 from flight_club.auth.views import login_required
 from flight_club.sessions.fc_sessions import FCSession
+from werkzeug.exceptions import BadRequestKeyError
 
+import datetime
 import flight_club.models.db_func as db_func
 
 bp = Blueprint("sessions", __name__, url_prefix="/sessions")
 
 
+def session_input_validator(session_id, date, host):
```

Comment:
I won't say this is needed, but I'd recommend trying out [type hinting](https://docs.python.org/3/library/typing.html) for python code.  Especially for something like this that returns some crazy tuple thing

Reply:
Yeah I can add the type hinting. I've just been lazy.

Reply:
We could alternatively re-write the whole app in a statically typed language like Rust.  But I'm guessing that's more work and wouldn't be appreciated since you've written 99% of it all :)

Reply:
I'll rewrite it all in C. On it.

Reply:
I hard agree with type hinting, that's also something I like to be extra anal about and capture doubly in the docstrings (ie `session_id (int)` and `data (datetime)`), it makes navigating big repos in vscode easier with nice and informative hover text. But the type hinting alone should be enough